tools/omnissm/cmd/omnissm/main.go

```diff
@@ -43,6 +43,9 @@ func checkDebug(cmd *cobra.Command, args []string) {
 func init() {
 	viper.AutomaticEnv()
 	viper.SetEnvPrefix("OMNISSM")
+	viper.SetConfigName("config")
```

Comment:
config is a bit generic for the filename prefix, would suggest omnissm, especially if we're picking up currently working dir as a location, also works better with the datastore options on viper which use this as a namespace.